synapse/push/httppusher.py

```diff
@@ -27,6 +27,7 @@
 from synapse.metrics.background_process_metrics import run_as_background_process
 from synapse.push import Pusher, PusherConfig, PusherConfigException
 
+from ..storage.databases.main.event_push_actions import HttpPushAction
```

Comment:
```suggestion
from synapse.storage.databases.main.event_push_actions import HttpPushAction
```
absolute imports are preferred in synapse